### PR TITLE
feat: add User and Otp entities

### PR DESCRIPTION
**Pull request by elitekaycy**
1. added user models
2. configured postgres set up

### CODE REVIEW

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# PostgreSQL database configuration
+
+spring.datasource.url=jdbc:postgresql://${DB_HOST}:${DB_PORT}/${DB_NAME}
+spring.datasource.username=${DB_USERNAME}
+spring.datasource.password=${DB_PASSWORD}
+
+# Hibernate properties
+spring.jpa.hibernate.ddl-auto=update
+spring.jpa.show-sql=true
+secret-key=8ffb0df0e3db0551b3b2b9fa3dad501a6582e3a3d7c82a53f4273c786492922c
```
Comment (Gabriel-Rockson) [inline src/main/resources/application.properties]: I will rather take this secret key out of here and have it in an environment variable

File: src/main/java/com/amalitech/gpuconfigurator/model/Otp.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity
+@Table(name = "otp")
+public class Otp {
+
```
Comment (Gabriel-Rockson) [inline src/main/java/com/amalitech/gpuconfigurator/model/Otp.java:28]: It'll be nice to have the fields createdAt, updatedAt, and deletedAt on the models for future data integrity

**Review by Gabriel-Rockson — CHANGES_REQUESTED** [review]
You can take a look at the conventional commits https://www.conventionalcommits.org/en/v1.0.0/ to make your github commit messages better.

File: src/main/resources/application.properties (hunk 1 above)
Comment (elitekaycy) [reply src/main/resources/application.properties]: alright

File: src/main/java/com/amalitech/gpuconfigurator/model/Otp.java (hunk 2 above)
Comment (elitekaycy) [reply src/main/java/com/amalitech/gpuconfigurator/model/Otp.java:28]: alright

**Review by Gabriel-Rockson — APPROVED** [review]

**Review by Gabriel-Rockson — APPROVED** [review]

**Comment by Gabriel-Rockson** [comment]
@elitekaycy You can merge this, and get the other PR updated, then I'd take a look at that.

**Pull request merged**